index.html

```diff
@@ -49,6 +49,212 @@ <h1 class="pad-top-158 font-size-40 font-weight-800 line-height-52 slategray mar
             <li><a href="#" class="margin-left-16"><img src="/assets/angelist.svg" alt="angellist icon"></a></li>
         </ul>
     </section>
+
+    <div class="grid d-grid">
```

Comment:
- [ ] The container of the works section should be a `<section>`  tag not a `<div>` tag. Please correct this.

Reply:
Thanks for pointing that out it has been changed.